model/resource/oci.yaml

```
@@ -0,0 +1,22 @@
+groups:
+  - id: oci
+    prefix: oci
+    type: resource
+    brief: >
+      An OCI instance.
+    attributes:
+      - id: manifest.digests
```

```suggestion
      - id: digest
```
The property is `digest` singular, right? https://github.com/opencontainers/image-spec/blob/main/descriptor.md#digests

> The value of the `digest` property is a string consisting of an algorithm portion and an encoded portion